universal-application-tool-0.0.1/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -26,6 +28,7 @@
 
 public class TrustedIntermediaryController {
 
+  private static final int PAGE_SIZE = 10;
```

Comment:
suggestion (non-blocking): bigger page size? maybe 30? i guess as a user i usually crank the page size up to whatever the max setting is

Reply:
The page is already super busy and with the search feature I figure it's probably better to make this window small - there's stuff below it, still.  If, given that, you still want it bigger, lmk and I'll do it.